### PR TITLE
[CallWithChat] Copy Invite Link margin added and sidepane cancel button border radius styled

### PR DESCRIPTION
**Pull request by edwardlee-msft**
# What
[CallWithChat] Copy Invite Link margin added and sidepane cancel button border radius styled

# Why
Copy Invite link was too close to the edge of the browser. 
https://skype.visualstudio.com/SPOOL/_workitems/edit/2777071
Cancel button hover highlight border radius did not match other button styles.
https://skype.visualstudio.com/SPOOL/_workitems/edit/2777072

# How Tested
Tested locally on CallWithChat Sample

# Process & policy checklist
- [x] I have updated the project documentation to reflect my changes if necessary.
- [x] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

**Review by JamesBurnside — APPROVED** [review]

File: packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx [hunk 1]
```diff
@@ -131,7 +131,12 @@ export const EmbeddedPeoplePane = (props: {
     >
       <Stack tokens={peoplePaneContainerTokens}>
         {inviteLink && (
-          <DefaultButton text="Copy invite link" iconProps={{ iconName: 'Link' }} onClick={() => copy(inviteLink)} />
+          <DefaultButton
+            text="Copy invite link"
```
Comment (alkwa-msft) [inline packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx:135]: should there be a localized string used here instead?

**Review by alkwa-msft — COMMENTED** [review]
Should we be using a localized string instead of the english string here?

**Comment by anjulgarg** [comment]
Looks like an icon in messed up after code changes
image

**Comment by edwardlee-msft** [comment]
> Looks like an icon in messed up after code changes image

That's odd as my diff shows the icon and my locally ran sample shows the icon as well. 
Do you still have the icon missing @anjulgarg ?
image

File: packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx (hunk 1 above)
Comment (edwardlee-msft) [reply packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx]: Resolved by adding localized string:
text={callWithChatStrings.copyInviteLinkButtonLabel}

**Review by dmceachernmsft — APPROVED** [review]

**Comment by dmceachernmsft** [comment]
> > Looks like an icon in messed up after code changes image
> 
> That's odd as my diff shows the icon and my locally ran sample shows the icon as well. Do you still have the icon missing @anjulgarg ? image

I believe this is a timeout issue with the update snapshots CI, I am seeing it in my PR's as well but depending on the run it will be correct. its happening to me in this PR: https://github.com/Azure/communication-ui-library/pull/1579

**Review by alkwa-msft — APPROVED** [review]

**Review by mgamis-msft — APPROVED** [review]

**Comment by mgamis-msft** [comment]
> > > Looks like an icon in messed up after code changes image
> > 
> > 
> > That's odd as my diff shows the icon and my locally ran sample shows the icon as well. Do you still have the icon missing @anjulgarg ? image
> 
> I believe this is a timeout issue with the update snapshots CI, I am seeing it in my PR's as well but depending on the run it will be correct. its happening to me in this PR: #1579

Yes, I have seen this in my PR too. Filed this bug https://skype.visualstudio.com/SPOOL/_workitems/edit/2782334. This issue is not related to the changes in this PR

**Pull request merged**